client/src/profile/Profile.js

```diff
@@ -1,8 +1,15 @@
 import React from 'react';
+import AddressField from './AddressField';
+
+function tmpStoreAddressObject(address) {
```

Comment:
This should probably be passed down as a prop (and there is a current PR that adds a ProfileContainer.js component)